### ISSUE TITLE
Suggestion: Amethyst Shard instead of Glass Pane

### ISSUE THREAD

**Issue by YumatanGames** [ChloeDawn/Frameless#1]
Hi! I had a very similar idea the other day about a mod that would do the exact same thing. The only difference is that instead of a glass pane, I though to use an Amethyst shard. My line of thought was that it would make it even better to find a Geode and give it another reason to find one. Since amethysts are renewable, new and kinda magical, I think it's a better option. What do you think? 😄

**Comment by ChloeDawn** [comment]
I agree that an amethyst shard would be a good option for this mechanic, however, I will likely not change the item as it would result in disparity for existing worlds (frames previously made invisible with a glass pane now dropping amethyst shards). I may look into making it configurable for those who want to change it.

**Comment by YumatanGames** [comment]
My Initial idea was to consume the amethyst shard. Like it was really used to make the frame invisible through some sort of magic.. So breaking the frame would drop only the normal item frame, and to make it invisible again you would need another shard. An option to use that mechanic instead of the glass pane would be nice. Anyway, thank you for your mod, and for considering my suggestion! =)
